datastr/dict.go

```diff
@@ -111,9 +124,13 @@ func (d *Dict) expandIfNeeded() {
 	}
 }
 
-func split(key [16]byte) (uint64, uint64) {
+func (d *Dict) split(key [16]byte) (uint64, uint64) {
+	if len(key) == 0 || len(key) < 16 {
```

Comment:
len(key) is always 16 because key is a [16]byte so this check is useless. the same for the line 133.

Reply:
true, this check was left from when it wasn't